toolkits/global/packages/global-loader-animation/scss/50-components/_loader.scss

```diff
@@ -0,0 +1,73 @@
+@import '../10-settings/default';
```

Comment:
You don't need to do this, they get imported separately in the app. where you have `// Include this with your other components` in the README, just include a line before the component import that imports the settings file